README.md

```diff
@@ -1,4 +1,89 @@
-[![GitHub Actions CI/CD build status — Collection test suite](https://github.com/ansible-collection-migration/community.crypto/workflows/Collection%20test%20suite/badge.svg?branch=master)](https://github.com/ansible-collection-migration/community.crypto/actions?query=workflow%3A%22Collection%20test%20suite%22)
+# Foo Collection
```

Comment:
```suggestion
# Crypto Collection
```

Reply:
LOL, I missed that! I accidentally closed the README before saving, and had to do most of it again, and apparently overlooked this part ;-)

Reply:
I've changed it to `# Ansible Community Crypto Collection`.